src/LocalInstaller.ts

```diff
@@ -71,7 +71,10 @@ export class LocalInstaller extends EventEmitter {
 
     private installOne(target: InstallTarget): Promise<void> {
         const toInstall = target.sources.map(source => resolvePackFile(source.packageJson)).join(' ');
-        const options: ExecOptions = { cwd: target.directory };
+        const options: ExecOptions = {
+            cwd: target.directory,
+            maxBuffer: 1024 * 1024 * 10
```

Comment:
Could we replace this with a constant? 

```ts
const TEN_MEGA_BYTE = 1024 * 1024 * 10;
```

Can be placed at the top of the file and reused where needed.